### DISCUSSION TITLE
Where to get client_id for SQLab APIs?

### DISCUSSION THREAD

**Discussion by EuphoriaCelestial** [apache/superset#28096] [Q&A / Help]
Hi, I saw that `/api/v1/sqllab/execute/` require a field named `client_id`, but I don't know how to get that ID. Can someone please tell me? Thanks a lot

**Comment by jitesh-raut** [comment]
Hi,

You don't need to pass the `client_id` when calling `/api/v1/sqllab/execute/`, as it will be provided in the API response. If you still wish to include it, you can use any 10-character string. Note that this string cannot be reused in subsequent executions.

Thanks!
